app/src/main/java/org/thoughtcrime/securesms/preferences/SettingsActivity.kt

```diff
@@ -394,18 +396,20 @@ class SettingsActivity : PassphraseRequiredActionBarActivity() {
                 horizontalArrangement = Arrangement.spacedBy(16.dp),
             ) {
                 OutlineButton(
+                    stringResource(R.string.share),
                     modifier = Modifier.weight(1f),
                     onClick = { sendInvitationToUseSession() }
-                ) {
-                    Text(stringResource(R.string.share))
-                }
+                )
 
                 OutlineTemporaryStateButton(
```

Comment:
This component is used multiple times and only ever adds a Text as its content..
Couldn't this be simplified by simply taking in a string, so it can internally rely on the existing OutlineButton which already sets the common style for our buttons?

Reply:
done